tests/drivers/w1/w1_api/testcase.yaml

```diff
@@ -1,9 +1,12 @@
 common:
   tags: drivers w1 userspace
   harness: ztest
-  harness_config:
-    fixture: w1_serial_idle
 tests:
   drivers.w1.w1-serial:
     filter: dt_compat_enabled("zephyr,w1-serial")
     platform_allow: nucleo_g0b1re nrf52840dk_nrf52840
+    harness_config:
+      fixture: w1_serial_idle
+  drivers.w1.w1-serial:
+    filter: dt_compat_enabled("zephyr,ds2484")
+    platform_allow: nrf52840dk_nrf52840
```

Comment:
Please review the test case configuration.
It seems as if you would have wanted to add a configuration for the device with extra_args, but only added a filter instead.

Reply:
Fixed